eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -657,18 +655,6 @@ public void forEachValue(Procedure<? super V> procedure)
         return new HashBiMap<>(this.delegate.groupByUniqueKey(function));
     }
 
-    @Override
```

Comment:
Similar to the other pull request that was pulling up default methods, I would prefer to remove only purely structural code.